index.js

```diff
@@ -1,17 +1,48 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  updateLength() {
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.updateLength()
+  }
 
-  max() {}
+  get(pos) {
+    if (!this.items[pos]) throw new Error('OutOfBounds');
 
-  min() {}
+    return this.items[pos]
```

Comment:
Aqui vc pode retornar um ternário:

return this.items[pos] ? this.items[pos] : throw new Error('OutOfBounds');

Fica legal assim :-)